tests/test_utils.py

```diff
@@ -124,6 +124,7 @@ def test_pil_version(self):
             5.4.1 no
```

Comment:
→ yes?

Reply:
Test does **not** fail in 5.4.1

Reply:
Oh. I thought it reads “the test **should** fail” (in the sense of *you may not use this version*).